src/scenes/home/families/facts/facts.css

```diff
@@ -1,26 +1,54 @@
-
 .facts {
-  display: 'flex';
-  background-color: '#E2E2E2';
-  align-items: 'center';
-  padding: '96px';
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 50px 0;
+}
+
+@media screen and (max-width: 1024px) {
+  .facts {
+    display: block;
+  }
 }
-.factsHeading {
-  font-size: '50px';
-  text-align: 'center';
+
+.factsHeading h2 {
+  width: 100%;
 }
+
 .factsList {
-  color: '#445366';
-  font-size: '24px';
-  padding: '0 32px';
+  flex: .48;
+  color: #445366;
+  font-size: 1.5rem;
+}
+
+.factsList ul {
+  width: 80%;
+  margin: 0 auto;
 }
+
+.factsList .footnote {
+  font-size: 80%;
+  font-style: italic;
+  margin: 0 auto;
+  text-align: left;
+  width: 75%;
+}
+
 .factsRight {
```

Comment:
I didn't want to give you more lame instructions on styles, so I made some final changes I'd like you to commit:

```
.factsRight {
  background-position: 70% 30%;
  border-radius: 40px;
  flex: 0.48;
  height: 50vh;
  margin-top: 65px;
  overflow: hidden;
}

.factItem {
  padding-bottom: 24px;
}

/* most mobile browsers have annoying behavior with vh on scroll */
@media screen and (max-width: 420px) {
  .factsRight {
    height: 300px;
  }
}
```

I changed background-position simply because it seems to work well on my iPhone, laptop, and desktop with that setting.